netbox/templates/core/plugin.html

```diff
@@ -0,0 +1,154 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load form_helpers %}
+{% load i18n %}
+
+{% block title %}{{ plugin.title_short }}{% endblock %}
+
+{% block object_identifier %}
+{% endblock object_identifier %}
+
+{% block breadcrumbs %}
+  <li class="breadcrumb-item"><a href="{% url 'core:plugin_list' %}">{% trans "Plugins" %}</a></li>
+{% endblock breadcrumbs %}
+
+{% block subtitle %}
+  <div class="text-secondary fs-5">
+    {{ plugin.tag_line }}
+    <a href="{{ plugin.homepage_url }}" target="_blank">Learn more <i class="mdi mdi-launch"></i></a>
+  </div>
+{% endblock subtitle %}
+
+{% block controls %}
+{% endblock %}
+
+{% block tabs %}
+  <ul class="nav nav-tabs" role="tablist">
+    <li class="nav-item" role="presentation">
+      <a class="nav-link active" id="overview-tab" data-bs-toggle="tab" data-bs-target="#overview" type="button" role="tab" aria-controls="edit-form" aria-selected="true">
+        {% trans "Overview" %}
+      </a>
+    </li>
+    {% if not plugin.is_local %}
+      <li class="nav-item" role="presentation">
+        <button class="nav-link" id="version-history-tab" data-bs-toggle="tab" data-bs-target="#version-history" type="button" role="tab" aria-controls="object-list" aria-selected="false">
+          {% trans "Version history" %}
+        </button>
+      </li>
+      {% if 'commercial' not in settings.RELEASE.features %}
+        <li class="nav-item" role="presentation">
+          <button class="nav-link" id="install-tab" data-bs-toggle="tab" data-bs-target="#install" type="button" role="tab" aria-controls="object-list" aria-selected="false">
+            {% trans "Install" %}
+          </button>
+        </li>
+      {% endif %}
+    {% endif %}
+  </ul>
+
+{% endblock tabs %}
+
+{% block content %}
+
+  <div class="tab-pane show active" id="overview" role="tabpanel" aria-labelledby="overview-tab">
+
+  <div class="row">
+    <div class="col col-md-12">
+
+      <div class="card">
+        <h5 class="card-header">{% trans "Author" %}</h5>
```

Comment:
Because the `org_id` appears to be a non-user-facing field, and `url` currently seems to be tethered to `plugin.homepage_url`, let's just move the author info to a row in the _Plugin_ table. If `author.url` gets a life of its own down the road, we can linkify the author name with it.